OpenSim/Common/Component.h

```diff
@@ -1475,9 +1475,11 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Component, Object);
     list their subcomponents, and so on.                                      */
     void printSubcomponentInfo() const;
     
-    /** List all the Sockets and Inputs and whether or not they are
-     * connected. */
-    void dumpConnections() const;
+    /** List all the Sockets and whether or not they are connected. */
```

Comment:
Could you also add to the documentation the fact that these functions list the names of the connectees, if the socket/input is connected?